dev/com.ibm.ws.org.apache.cxf.cxf.rt.transports.http.2.6.2/src/org/apache/cxf/transport/http/Headers.java

```diff
@@ -370,12 +372,8 @@ private boolean isResponseBodyAvailable() {
         if (ctLen == null || ctLen.size() != 1) {
             return true;
         }
-        try {
-            if (Integer.valueOf(ctLen.get(0)) == 0) {
-                return false;
-            }
-        } catch (NumberFormatException ex) {
-            // ignore
+        if ("0".equals(ctLen.get(0))) {
+            return false;
```

Comment:
I was thinking about this and a String of "000" is also zero, so may need to still have the  parsing of the String.  In reality it was calling Integer.valueOf in the old code when it should be Integer.parseInt instead.